evaluator/src/main/kotlin/Evaluator.kt

```diff
@@ -49,11 +49,8 @@ class Evaluator(
         scriptsInstancesSharing(true)
     }
 
-    fun run(script: String): EvaluatorRun {
-        val startTime = Instant.now()
+    fun run(script: String): List<RuleViolation> {
```

Comment:
I would prefer if the return type would stay `EvaluatorRun` because it is better for reusability of the function, otherwise if the function is used programmatically (e.g. in the upcoming server) the code to generate the `EvaluatorRun` needs to be duplicated. This would become more of an issue if more properties like `options` or `environment` would be added to `EvaluatorRun` later.
Maybe instead of calling this function multiple times, change the function to take multiple scripts?

Reply:
> Maybe instead of calling this function multiple times, change the function to take multiple scripts?

Good idea, done.